tensorflow/lite/micro/examples/hello_world/models/BUILD

```diff
@@ -0,0 +1,24 @@
+load("//tensorflow/lite/micro:build_def.bzl", "generate_cc_arrays")
+
+package(
+    default_visibility = ["//visibility:public"],
```

Comment:
remove default public visibility.

The targets that do need visibility already have the correct visibility.

Created a related bug for at least one additional instance of public visibility: http://b/270650627

Reply:
done